static/app/views/performance/landing/widgets/components/accordion.tsx

```diff
@@ -86,3 +110,11 @@ const ListItemContainer = styled('div')`
 const StyledContentContainer = styled('div')`
   padding: ${space(0)} ${space(2)};
 `;
+
+const ReplayContentContainer = styled('div')`
+  padding: ${space(0)} ${space(0.25)};
+`;
+
+const StyledButton = styled(Button)`
+  margin-right: ${space(1.5)};
```

Comment:
👀

Reply:
i couldn't figure out how to avoid this one or maybe my morning brain is on.....

Reply:
if `<LeftContentContainer>` had padding there... but it would be better if the `{content}` itself decided how far away to be.

`<RightAlignedCell>`, which holds the icon and the count, had it's own padding to keep it separate from the accordian button when the button was on the right side. That padding could probably go away.

Reply:
Like, if the content decides it's own padding, then we wouldn't need `StyledContentContainer` and `LeftContentContainer`

Reply:
removed the padding from `RightAlignedCell` and put a `column-gap` inside the `ButtonLeftListItemContainer` inside `Accordion`